client/src/components/table/Table.js

```diff
@@ -0,0 +1,82 @@
+import React, { useState } from 'react';
+import { AgGridReact } from 'ag-grid-react';
+
+import 'ag-grid-community/dist/styles/ag-grid.css';
+import 'ag-grid-community/dist/styles/ag-theme-alpine.css';
+
+const Table = () => {
+  const [columnDefs] = useState([
+    { headerName: 'Name', field: 'name', width: 100, sortable: true },
+    { headerName: 'Email', field: 'email', width: 180, sortable: true },
+    { headerName: 'Phone #', field: 'phone', width: 160, sortable: true },
+    { headerName: 'NetID', field: 'netid', width: 120, sortable: true },
+    { headerName: 'UIN', field: 'uin', width: 120, sortable: true },
+    { headerName: 'Major', field: 'major', width: 120, sortable: true },
+    {
+      headerName: 'Birth Date',
+      field: 'birthdate',
+      width: 120,
+      sortable: true,
+    },
+    { headerName: 'Github', field: 'github', width: 140, sortable: true },
+    { headerName: 'Snapchat', field: 'snapchat', width: 140, sortable: true },
+    { headerName: 'Instagram', field: 'instagram', width: 140, sortable: true },
+    { headerName: 'Grad Sem/Yr', field: 'grad', width: 190, sortable: true },
+    {
+      headerName: 'Generation',
+      field: 'generation',
+      width: 120,
+      sortable: true,
+    },
+    { headerName: 'Location', field: 'location', width: 120, sortable: true },
+    { headerName: 'Role', field: 'role', width: 160, sortable: true },
+    { headerName: 'Level', field: 'level', width: 120, sortable: true },
+    { headerName: 'Status', field: 'status', width: 120, sortable: true },
+    { headerName: 'Dues', field: 'dues', width: 100, sortable: true },
```

Comment:
Some of these fields are not going to be visible to all people (right now I'm thinking it'll just be level and dues), so we will have to conditionally render columns depending on what data is returned from the backend. You could add some logic for this now, or we could hold off until we actually connect this to the endpoint, whatever you think is best.

Reply:
Yea my bad, I really shouldn't have touched this portion of the project in this PR, just added extra confusion. I haven't actually touched any part of the RBAC yet and the purpose of this PR really didn't require it, so I will probably just abstract that process to a different PR if that is okay with you.

Reply:
Sounds good